core/registry.ts

```diff
@@ -316,18 +294,20 @@ export function getObject<T>(
  * Returns a map of items registered with the given type.
  *
  * @param type The type of the plugin. (e.g. Category)
- * @param opt_cased Whether or not to return a map with cased keys (rather than
- *     caseless keys). False by default.
+ * @param opt_cased Deprecated.
  * @param opt_throwIfMissing Whether or not to throw an error if we are unable
  *     to find the object. False by default.
  * @returns A map of objects with the given type, or null if none exists.
  */
 export function getAllItems<T>(
   type: string | Type<T>,
-  opt_cased?: boolean,
+  opt_cased?: any,
```

Comment:
Unfortunately after looking into it I'm no longer sure this is safe to remove, at least not without doing some work to make the serializer case-insensitive.

If someone registered a serializer with an uppercase name, then their existing serialized workspace data would have uppercase characters in it. After this change, if we tried to deserialize that data, the names wouldn't match and the data wouldn't load correctly. @BeksOmega can you confirm that? Would this be safe if we changed serialization to also lowercase the state keys when loading?

Reply:
Yeah I believe if someone has registered a serializer with capitals (which is would be reasonable if they were using `camelCase`) their data will no longer load. Should be relatively easy to add a test for that.

Given that, I have a small opinion that this is not worth changing.